backend/orchestrators/Orchestrator.py

```diff
@@ -92,6 +100,12 @@ def conversation_without_data(self, request_body, message_uuid):
 
     message_uuid = ""
 
+    conversation_client = ConversationTelemetryClient(cosmosdb_endpoint=str(AZURE_COSMOSDB_ENDPOINT), 
```

Comment:
Instantiating the conv_client in the base orchestrator would seem to place a required dependency on having a CosmosDB configured. I think the original upstream repo doesn't require that. How does this line (and usages of the conv_client elsewhere) behave if no CosmosDB is configured?

Reply:
I wouldn't say it's blocking us. Will definitely need to be addressed before attempting to PR back upstream though. But to go live it's not a concern.